src/com/palmergames/bukkit/towny/listeners/TownyPlayerListener.java

```diff
@@ -585,6 +586,7 @@ public void onPlayerMove(PlayerMoveEvent event) {
 		Location from;
 		PlayerCache cache = plugin.getCache(player);
 		Resident resident = null;
+
```

Comment:
Whitespacing change, please revert.

Reply:
fixing... not even sure how that happened.